recipe/meta.yaml

```diff
@@ -19,23 +19,25 @@ requirements:
     - {{ compiler('c') }}
   host:
     - cython >=0.29,!=0.29.18,!=0.29.31
-    - numpy   1.21  # [py<311]
-    - numpy   1.23  # [py==311]
+    - numpy {{ numpy }}
     - pip
     - python
     - setuptools
     - wheel
+    - packaging
```

Comment:
wondering why scipy and statsmodels aren't here and packaging is? as per https://github.com/alkaline-ml/pmdarima/blob/v2.0.4/pyproject.toml

Reply:
`scipy` and `statsmodels` are not uin the host because they are not needed to build the project. We don't know why they have them in their build dependencies. As for `packaging`, they explicitly import `packaging` in `setup.py`, see https://github.com/alkaline-ml/pmdarima/blob/v2.0.4/setup.py#L15. They probably get it from a transitive dependencies (I think from statsmodels) and it's a bug that they don't specify it in their build deps.

Reply:
ok thanks. I guess if you had put in the deps as per the pyproject.toml, `scipy` and `statsmodels` would be caught by an overdepending check, and `packaging` by an error running setup.py? (or maybe an overlinking check if it gets pulled in by something else)?
Means, it's often useful to build with `--error-overdepending` during development, even if rocket-platform doesn't enable it by default

Reply:
> guess if you had put in the deps as per the pyproject.toml, scipy and statsmodels would be caught by an overdepending check

Not in this case. overdepending and overlinking are for runtime deps and not only that, but they are for shared libraries. `scipy` and `statsmodels` are python packages. They contain shared libraries (Python extensions) but conda-build wouldn't consider them for the overdepending check.

Reply:
ok, thanks.